x-pack/test/screenshot_creation/apps/ml_docs/data_frame_analytics/outlier_detection.ts

```diff
@@ -0,0 +1,132 @@
+  const ml = getService('ml');
+  const mlScreenshots = getService('mlScreenshots');
+  const transform = getService('transform');
+
+  const screenshotDirectories = ['ml_docs', 'data_frame_analytics'];
+
+  const transformConfig = {
+    id: `logs-by-clientip`,
+    source: { index: LOGS_INDEX_PATTERN },
+    pivot: {
+      group_by: { clientip: { terms: { field: 'clientip' } } },
+      aggregations: {
+        '@timestamp.value_count': { value_count: { field: '@timestamp' } },
+        'bytes.max': { max: { field: 'bytes' } },
+        'bytes.sum': { sum: { field: 'bytes' } },
+        'request.value_count': { value_count: { field: 'request.keyword' } },
+      },
+    },
+    description: 'Web logs by client IP',
+    dest: { index: 'weblog-clientip' },
+  };
+
+  const outlierJobConfig: DeepPartial<DataFrameAnalyticsConfig> = {
+    id: 'weblog-outliers',
+    source: { index: 'weblog-clientip' },
+    dest: { index: 'weblog-outliers', results_field: 'ml' },
+    analysis: { outlier_detection: {} },
+    analyzed_fields: {
+      includes: ['@timestamp.value_count', 'bytes.max', 'bytes.sum', 'request.value_count'],
+      excludes: [],
+    },
+    model_memory_limit: '20mb',
+  };
+
+  describe('outlier detection job', function () {
+    before(async () => {
+      await transform.api.createAndRunTransform(transformConfig.id, transformConfig);
+      await ml.testResources.createIndexPatternIfNeeded(transformConfig.dest.index);
+
+      await ml.api.createAndRunDFAJob(outlierJobConfig as DataFrameAnalyticsConfig);
+      await ml.testResources.createIndexPatternIfNeeded(outlierJobConfig.dest!.index!);
+    });
+
+    after(async () => {
+      await ml.testResources.deleteIndexPatternByTitle(transformConfig.dest.index);
+      await transform.api.deleteIndices(transformConfig.dest.index);
+      await transform.api.cleanTransformIndices();
+
+      await ml.api.deleteDataFrameAnalyticsJobES(outlierJobConfig.id as string);
+      await ml.testResources.deleteIndexPatternByTitle(outlierJobConfig.dest!.index!);
+      await ml.api.deleteIndices(outlierJobConfig.dest!.index!);
+      await ml.api.cleanMlIndices();
+    });
+
+    it('transform screenshot', async () => {
+      await ml.testExecution.logTestStep('navigate to transform list');
+      await transform.navigation.navigateTo();
+
+      await transform.testExecution.logTestStep('open transform in wizard');
+      await transform.management.assertTransformListPageExists();
+      await transform.table.refreshTransformList();
+      await transform.table.filterWithSearchString(transformConfig.id, 1);
+      await transform.table.assertTransformRowActions(transformConfig.id, false);
+      await transform.table.clickTransformRowAction(transformConfig.id, 'Clone');
+      await transform.wizard.assertDefineStepActive();
+
+      await ml.testExecution.logTestStep('take screenshot');
+      await mlScreenshots.takeScreenshot('logs-transform-preview', screenshotDirectories);
```

Comment:
Nit - might be worth including `clone` in the file name here, in case we add in a screenshot of the preview from  the usual create workflow e.g `logs-clone-transform-preview`?

Reply:
The screenshot file names are currently defined by the docs team and we're using the exact same names here, so it's easy for them to compare contents. I wouldn't change that as part of this PR, but we can discuss this with docs folks for a follow-up.